scripts/lib/aws/elasticache.sh

```diff
@@ -12,6 +12,29 @@ SCRIPTS_DIR="$ROOT_DIR/scripts"
 # functions for tests
 #################################################
 
+# k8s_controller_apply_temp_creds adds AWS temporary credentials
+# to service controller running on kubernetes cluster.
+# it depends upon:
+#   $service_name
+#   $AWS_ACCESS_KEY_ID
+#   $AWS_SECRET_ACCESS_KEY
+#   $AWS_SESSION_TOKEN
+k8s_controller_apply_temp_creds() {
```

Comment:
You already opened a PR for this commit cad9d79bf5e2d2d58ec2f726a06cc3c8daf8bbaf :)

Reply:
That PR was pending merge. To keep changes separate, I kept the commits as is in this PR and did not squash them.

Reply:
Ohh,  didn't know Github can do a strategic merge :O

Reply:
> Ohh, didn't know Github can do a strategic merge :O

it can't. basically, when the base PR is merged, then refreshing this page will remove the diff of the base patch's changes.

Reply:
As I updated PR #411, I have removed corresponding commit from this PR. Now, It contains only commit relevant to it.